src/javascripts/helpers/validationForm.js

```diff
@@ -0,0 +1,52 @@
+import { REGEX } from "../constants/config";
+import { MESSAGE } from "../constants/message";
+
+export const validationForm = (data) => {
+    const emailError = document.querySelector(".error-email");
+    const passwordError = document.querySelector(".error-password");
+
+   isValid = true;
+
+   const fields = [
```

Comment:
This config should be in constant or configuration for form, so that we can reuse it.